backend/capellacollab/models/crud.py

```diff
@@ -0,0 +1,75 @@
+# Standard library:
+import typing as t
+
+# 3rd party:
+from sqlalchemy.orm import Session
+
+# 1st party:
+import capellacollab.projects.crud as projects_crud
+from capellacollab.models.models import EmptyModel, Model, NewModel, ToolDetails
+from capellacollab.projects.models import DatabaseProject
+from capellacollab.tools.models import Tool, Type, Version
+
+
+def get_all(db: Session, project_slug: str) -> t.List[Model]:
+    project = (
+        db.query(DatabaseProject).filter(DatabaseProject.slug == project_slug).first()
+    )
+    assert project is not None
+    return db.query(Model).filter(Model.project_id == project.id).all()
+
+
+def get_id(db: Session, id_: int) -> Model:
```

Comment:
Same here, because you return no id, but a model. So, I would rename it to `get_model_by_id`